### PR TITLE
WIP. pj-17-category-page

### PR DESCRIPTION
**Pull request by NykoEugen**

### CODE REVIEW

File: categories/views.py [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
 
 def category_list(request: HttpRequest) -> HttpResponse:
     objects = MainCategory.objects.prefetch_related("category_set").all()
 
     return render(request, "categories/category_list.html", {"objects": objects})
+
+
+def category_detail(request: HttpRequest, category_id: int) -> HttpResponse:
+    objects = Post.objects.filter(category_id=category_id, is_active=1)
+    category = Category.objects.filter(pk=category_id)
```
Comment (acman) [inline categories/views.py]: better always check id or return 404
```suggestion
    category = get_object_or_404(category_id)
    posts = Post.objects.filter(category_id=category.id, is_active=True)
```

File: templates/categories/category_detail.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  Forum
+{% endblock %}
+
+{% block body %}
+  <div class="container">
+    <div class="section">
+      {% for item in category %}
```
Comment (acman) [inline templates/categories/category_detail.html]: I think there is only one category should be

**Review by acman — CHANGES_REQUESTED** [review]
Great start! some small fixes should be submitted

File: templates/categories/category_detail.html (hunk 2 above)
Comment (NykoEugen) [reply templates/categories/category_detail.html]: I could not display the attributes without for

File: categories/views.py (hunk 1 above)
Comment (NykoEugen) [reply categories/views.py]: category = get_object_or_404(category_id)   how it understand what i need if model doesn't specified

File: templates/categories/category_detail.html (hunk 2 above)
Comment (NykoEugen) [reply templates/categories/category_detail.html]: Done

File: categories/views.py (hunk 1 above)
Comment (NykoEugen) [reply categories/views.py]: Done)

File: templates/categories/category_detail.html (hunk 2 above)
Comment (acman) [reply templates/categories/category_detail.html]: yeah, filter returns queryset, that is why loop is required, but we expect one object here

**Review by acman — APPROVED** [review]
👍

**Pull request merged**